app/connectors/MessageConnector.scala

```diff
@@ -47,7 +48,9 @@ class MessageConnector @Inject()(config: AppConfig, http: HttpClient)(implicit e
 
     lazy val messageSender = MessageSender(arrivalId, message.messageCorrelationId)
 
-    val newHeaders = headerCarrier.withExtraHeaders(addHeaders(message.messageType, dateTime, messageSender): _*)
+    val newHeaders = headerCarrier
+      .copy(authorization = Some(Authorization("Bearer securityToken"))) //TODO placeholder, will be provided later
```

Comment:
We can remove the TODO for now since it's tracked by a ticket
```suggestion
      .copy(authorization = Some(Authorization("Bearer securityToken")))
```